tests/Feature/UserControllerTest.php

```diff
@@ -0,0 +1,569 @@
+<?php
+namespace Tests\Feature;
+
+use App\Http\Controllers\UserController as ControllerUser;
+use App\Http\Models\User as ModelUser;
+use App\Http\Response\Response;
+use Image;
+use Tests\TestCase;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Lang;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+final class UserControllerTest extends TestCase
+{
+	use DatabaseMigrations;
+
+	public function testAddUserSuccess()
+	{
+		$Response 		= new Response();
+    	$base_response	= $Response->response( '200', Lang::get('message.web.success.0001'), "", true );
+		$request_array 	= [
+            'name' 		=> 'abc',
+            'address' 	=> 'Test Dia Chi',
+            'age' 		=> 22,
+        ];
+        $response = $this->call('POST', 'user/add', $request_array);
+
+        // dd($response->getOriginalContent());
+        $this->assertDatabaseHas('users',
+            [
+                'name' 		=> $request_array['name'],
+                'age' 		=> $request_array['age'],
+                'address' 	=> $request_array['address'],
+            ]);
+        $response->assertJson( $base_response );
+	}
+
+	public function testAddUserFailWithConditionNameTooLong()
+	{
+		$Response 		= new Response();
```

Comment:
Please fix the indent

Reply:
In my sublime text, indent looks good @@